### PR TITLE
Remove some functions from ElasticSearchRestDAOV7.java which were mistakenly repeated

### PR DESCRIPTION
**Pull request by bjpirt**
Looks like there was a bad merge causing some of the methods in ElasticSearchRestDAOV7.java to be repeated that caused the build to fail

Pull Request type
----

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes (Please run `./gradlew generateLock saveLock` to refresh dependencies)
- [ ] WHOSUSING.md
- [ ] Other (please describe):

Changes in this PR
----

_Describe the new behavior from this PR, and why it's needed_
No new behaviour, but the build passes now

### CODE REVIEW

File: index/es7-persistence/src/main/java/com/netflix/conductor/es7/dao/index/ElasticSearchRestDAOV7.java [hunk 1]
```diff
@@ -1341,102 +1341,4 @@ BulkRequestWrapper getBulkRequest() {
             this.bulkRequest = new BulkRequestWrapper(bulkRequest);
         }
     }
-
-    @Override
-    public void removeTask(String workflowId, String taskId) {
-        try {
-            long startTime = Instant.now().toEpochMilli();
-            String docType = TASK_DOC_TYPE;
```
Comment (manan164) [inline index/es7-persistence/src/main/java/com/netflix/conductor/es7/dao/index/ElasticSearchRestDAOV7.java:1349]: Hi @bjpirt , Looks like this is the correct method. Ideally, we should be removing the other one. Can you please verify?

Comment (bjpirt) [reply index/es7-persistence/src/main/java/com/netflix/conductor/es7/dao/index/ElasticSearchRestDAOV7.java:1349]: I thought document types had been deprecated (as per https://github.com/Netflix/conductor-community/pull/170) and the calls in the functions I removed to `UpdateRequest` were displaying in my IDE as deprecated there, hence getting rid of these functions.

Comment (manan164) [reply index/es7-persistence/src/main/java/com/netflix/conductor/es7/dao/index/ElasticSearchRestDAOV7.java:1349]: Hi @james-deee and @bjpirt , Let me verify this.

Comment (manan164) [reply index/es7-persistence/src/main/java/com/netflix/conductor/es7/dao/index/ElasticSearchRestDAOV7.java:1349]: Hi @james-deee, @bjpirt , Changes looks fine. LGTM

**Pull request merged**